README.md

```diff
@@ -1,262 +1,327 @@
+
+The following example HTML/Jinja template can be found near the bottom of the [standalone_assessment.py](docs/quickstart/assessment/standalone_assessment.py) file.
+
+``` python
+        template = Template("""<!DOCTYPE html>
+        <html>
+            <head>
+                <link rel="stylesheet" type="text/css" href="../css/style.css">
+            </head>
+            <body>
+                <h1>{{ name }}</title></h1>
+                <!-- Items API will render the assessment app into this div. -->
+                <div id="learnosity_assess"></div>
+                <!-- Load the Items API library. -->
+                <script src=\"https://items.learnosity.com/?v2021.2.LTS/\"></script>
+                <!-- Initiate Items API assessment rendering, using the signed parameters. -->
+                <script>
+                    var itemsApp = LearnosityItems.init( {{ generated_request }} );
+                </script>
+            </body>
+        </html>
+        """)
 ```
 
-# Tests
+The important parts to be aware of in this HTML are:
 
-To run the tests, run `tox` from the project directory. This will run tests several different versions of python.
+* A div with `id="learnosity_assess"`. This is where the Learnosity assessment player will be rendered to deliver the assessment.
+* The `<script src="https://items.learnosity.com/?v2021.2.LTS"></script>` tag, which includes Learnosity's Items API on the page and makes the global `LearnosityItems` object available. The version specified as `v2021.2.LTS` will retrieve that specific [Long Term Support (LTS) version](https://help.learnosity.com/hc/en-us/articles/360001268538-Release-Cadence-and-Version-Lifecycle). In production, you should always pin to a specific LTS version to ensure version compatibility.
+* The call to `LearnosityItems.init()`, which initiates Items API to inject the assessment player into the page.
+* The variable `{{generated_request}}` dynamically sends the contents of our init options to JavaScript, so it can be passed to `init()`.
 
-If you don't have `tox` installed, run:
+The call to `init()` returns an instance of the ItemsApp, which we can use to programmatically drive the assessment using its methods. We pull in our Learnosity configuration in a variable `{{ generated_request }}`, that the Jinja template will import from the Python program. The variable `{{ name }}` is the page title which can be set in the same way.
 
-    pip install tox
+The Jinja template is rendered by the following line, which will bring in those variables.
 
-This assumes that you have `pyenv` or a similar tool set up to provide python binaries for 2.7, 3.3, 3.4, 3.5 and 3.6.
+``` python
+self.wfile.write(bytes(template.render(name='Standalone Assessment Example', generated_request=generated_request), "utf-8"))  
```

Comment:
Can we update this line to what's in the python file?

Reply:
Good pickup. Fixed here: https://github.com/Learnosity/learnosity-sdk-python/commit/9c8c48d829e0c61e5a638583b8a371663810399a